.github/workflows/vllm.yml

```diff
@@ -96,3 +96,7 @@ jobs:
         run: |
           pip3 install --upgrade vllm==0.8.3 tensordict==0.7.2
           pytest -svvv tests/workers/rollout/test_vllm_chat_scheduler.py
+      - name: Running LangGraph multi-turn rollout tests on 8 L20 GPUs
+        run: |
+          pip3 install -e .[langgraph]
+          pytest -svvv tests/workers/rollout/test_langgraph_chat_scheduler.py
```

Comment:
please see https://github.com/volcengine/verl/blob/main/.github/workflows/vllm.yml#L131

Reply:
@eric-haibin-lin The test uses vllm and multiple gpus. Should we add vllm and langgraph dependencies groups to gpu_unit_tests? Or would just renaming the test to include `test_vllm` be enough to avoid repeated tests?